From c9cb1e763bb9809a17e00187dd013a3b2ab7718c Mon Sep 17 00:00:00 2001
From: huangshuosheng <huangshuosheng@allwinnertech.com>
Date: Wed, 22 Apr 2020 11:32:27 +0800
Subject: [PATCH 746/830] ANDROID: gki_defconfig: enable
 CONFIG_PM_DEVFREQ_EVENT.
X-NVConfidentiality: public

Enable support for devfreq driver to adjust DDR frequency.

Bug: 154672404
Bug: 162909109
Change-Id: Ic42139c54c62c6ee1bea1da002af06088e2ef268
Signed-off-by: huangshuosheng <huangshuosheng@allwinnertech.com>
Signed-off-by: Connor O'Brien <connoro@google.com>
---
 arch/arm64/configs/gki_defconfig | 1 +
 arch/x86/configs/gki_defconfig   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index 34244fa08f97..b4f60cac5132 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -398,6 +398,7 @@ CONFIG_DEVFREQ_GOV_PERFORMANCE=y
 CONFIG_DEVFREQ_GOV_POWERSAVE=y
 CONFIG_DEVFREQ_GOV_USERSPACE=y
 CONFIG_DEVFREQ_GOV_PASSIVE=y
+CONFIG_PM_DEVFREQ_EVENT=y
 CONFIG_IIO=y
 CONFIG_IIO_BUFFER=y
 CONFIG_IIO_TRIGGER=y
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index ec1d7bd13c20..d25fc74db047 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -347,6 +347,7 @@ CONFIG_ION_SYSTEM_HEAP=y
 CONFIG_REMOTEPROC=y
 CONFIG_RPMSG_CHAR=y
 CONFIG_PM_DEVFREQ=y
+CONFIG_PM_DEVFREQ_EVENT=y
 CONFIG_IIO=y
 CONFIG_IIO_BUFFER=y
 CONFIG_IIO_TRIGGER=y
-- 
2.17.1

